app/celery/scheduled_tasks.py

```diff
@@ -46,6 +47,18 @@ def run_scheduled_jobs():
         raise
 
 
+@notify_celery.task(name='send-scheduled-notifications')
+@statsd(namespace="tasks")
+def send_scheduled_notifications():
+    try:
+        for notification in dao_get_scheduled_notifications():
```

Comment:
This could be gnarly - thats an unbounded action - could be thousands of API calls to SQS. Depending on how many things queued for scheduled sending. If this is for an overnight batch job - it could be 100,000 things ready to go.